disco_aws_automation/socify_helper.py

```diff
@@ -0,0 +1,84 @@
+"""
+This module has utility functions for working with the socify lambda
+"""
+import json
+import logging
+import requests
+from .disco_config import read_config
+
+logger = logging.getLogger(__name__)
+
+
+class SocifyHelper(object):
+    """Socify helper provides the function to invoke the Socify lambda functions"""
+    SOC_EVENT_OK = 100
+    SOC_EVENT_BAD_DATA = 200
+    SOC_EVENT_ERROR = 300
+
+    def __init__(self, ticket_id, command, sub_command=None, config=None):
+        self._ticket_id = ticket_id
+        self._command = command
+        self._sub_command = sub_command
+
+        if config:
+            self._config = config
+        else:
+            self._config = read_config()
+        self._socify_url = None
+
+    def _build_event_url(self):
+        """
+        Build the socify event url using the socify configuration data provided in the disco_aws.ini file
+        :return: The socify URL
+        """
+        if not self._socify_url:
+            self._socify_url = self._config.get("socify", "socify_baseurl")
```

Comment:
We should gracefully handle the case when `socify_baseurl` is missing. It will take some time for everyone to update their team configs to have it. We could initially log a warning and skip sending events to Socify if the url is missing